hub/api/tensor.py

```diff
@@ -58,19 +59,45 @@ def __init__(
 
             create_tensor(self.key, self.provider, tensor_meta)
 
-    def append(self, array: np.ndarray, batched: bool):
-        # TODO: split into `append`/`extend`
-        add_samples_to_tensor(
-            array,
-            self.key,
-            storage=self.provider,
-            batched=batched,
-        )
+    def extend(self, array: np.ndarray):
+        """Extend tensor by appending elements from a batched numpy array.
```

Comment:
Can we make this clearer?
Something like `Add multiple samples to a tensor in a single operation. This can be done by using a batched numpy array or a list of numpy arrays`